rotkehlchen/tests/data/expected/test_evm_transactions-test_query_transactions.json

```diff
@@ -1 +1 @@
659C251811CEC9c364ef91dC08D300C","value":"41000000000000000","gas":"274057","gas_price":"14688248714","gas_used":"226252","input_data":"0x5f575529000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000091a94863ca800000000000000000000000000000000000000000000000000000000000000000c0000000000000000000000000000000000000000000000000000000000000000f706d6d46656544796e616d69637634000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000003000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a0b86991c6218b36c1d19d4a2e9eb0ce3606eb480000000000000000000000000000000000000000000000000090943930da64000000000000000000000000000000000000000000000000000000000002e6cb6800000000000000000000000000000000000000000000000000000000000001200000000000000000000000000000000000000000000000000001150f32f01c00000000000000000000000000f326e4de8f66a0bdc0970b79e0924e33c79f1915000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001e0000000000000000000000000a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc20000000000000000000000000000000000000000000000000000000002f5f4210000000000000000000000000000000000000000000000000090943930da640000000000000000000000000056178a0d5f301baf6cf3e1cd53d9863437345bf900000000000000000000000074de5d4fcbf63e00296fd95d33236b9794016631000000000000000000000000b8553d9ee35dd23bb96fbd679e651b929821969b00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000063a7a24401ffffffffffffffffffffffffffffffffffffff33264e3863a7a1ae000000330000000000000000000000000000000000000000000000000000000000000003000000000000000000000000000000000000000000000000000000000000001ba36db1f4217e5d2fe99209742d5bbad549cb745940fd9ddc1ecfae06b1013207446649ed77f231e073e6a075082d5c2ace08858c50baa4071a28ad217216d22a0000000000000000000000000000000000000000000000000090943930da640039","nonce":0}},{"entry":{"tx_hash":"0xe32037154199c9b363aacbd6ce853fd4633cc62fdeb34a9305656ee280f4f6ef","evm_chain":"ethereum","timestamp":1671929963,"block_number":16258333,"from_address":"0x56Eddb7aa87536c09CCc2793473599fD21A8b17F","to_address":"0xb8553D9ee35dd23BB96fbd679E651B929821969B","value":"61577330000000000","gas":"207128","gas_price":"12391657059","gas_used":"21000","input_data":"0x","nonce":4031412}},{"entry":{"tx_hash":"0xfeabe8233f374eed505960f4bf8364a9d9398d7e1320ff7df425934134450231","evm_chain":"ethereum","timestamp":1671926939,"block_number":16258082,"from_address":"0x3794Da19a22CeDe7976f9d3181F2CCD58f6408a5","to_address":"0xb8553D9ee35dd23BB96fbd679E651B929821969B","value":"20000000000000000","gas":"21000","gas_price":"10625802284","gas_used":"21000","input_data":"0x","nonce":5}}],"entries_found":49,"entries_total":18,"entries_limit":100}
```

Comment:
Why was a change here needed? Perhaps it would make more sense to VCR the test instead?

Reply:
This changed because I also removed the flag from the ethereum transactions endpoint as discussed in discord. I believe it doesn't make sense to use VCR here since we are comparing the local output agaisnt the expected one in this file